packages/nextjs/app/builders/[0x575109e921C6d6a1Cb7cA60Be0191B10950AfA6C]/page.tsx

```diff
@@ -0,0 +1,106 @@
+import Image from "next/image";
+import { Metadata } from "next";
+
+export const metadata: Metadata = {
+  title: "Smartdev | Developer Profile",
+  description: "Personal profile page for Smartdev, a Solidity developer passionate about DeFi and DAO governance.",
+};
+
+export default function SmartdevProfilePage() {
+  return (
+    <main className="min-h-screen p-8 max-w-3xl mx-auto">
+      <section className="flex flex-col md:flex-row gap-8 items-center mb-12">
+        <div className="w-32 h-32 rounded-full bg-gray-200 overflow-hidden">
+          <Image
+            src="/profile-pic.jpeg"
+            alt="Smartdev Avatar"
+            width={128}
+            height={128}
+            className="w-full h-full object-cover"
+          />
+        </div>
+        <div>
+          <h1 className="text-3xl font-bold">Smartdev</h1>
+          <p className="text-lg text-gray-600 mt-2">0x575109e921C6d6a1Cb7cA60Be0191B10950AfA6C</p>
```

Comment:
There's a component in this project that renders the address. You should try using it.